app/components/user-skills-input.js

```diff
@@ -1,16 +1,28 @@
+import { EKMixin, keyDown, getCode } from 'ember-keyboard';
 import Ember from 'ember';
 
 const {
   Component,
   computed,
-  computed: { alias, and, equal, not, notEmpty },
+  computed: { alias, and, not, notEmpty },
   inject: { service },
   isEmpty,
   run: { once },
-  observer
+  observer,
+  on,
+  TextField
 } = Ember;
 
-export default Component.extend({
+Ember.TextField.extend(EKMixin, {
+  activateKeyboard: on('init', function() {
+    this.set('keyboardActivated', true);
+  })
+});
+
+export default Component.extend(EKMixin, {
+  activateKeyboard: on('init', function() {
```

Comment:
Rather than calling `on('init', function() { ... } )` I believe it is now recommended to explicitly overwrite `init` and call `this._super`.  This could be updated to:

```
init() {
  this.set('keyboardActivated', true);
  this._super(...arguments);
},
```

For reference: http://notmessenger.com/proper-use-of-ember-on/